CHANGELOG.md

```diff
@@ -23,23 +23,6 @@
 ### Fixed
 * Sync client may [have failed an assertion](https://github.com/realm/realm-core/blob/006660c8d20c4941d3838f74aec6f3561ebf6784/src/realm/sync/noinst/client_impl_base.cpp#L388) during shutdown if all sessions hadn't been ready to finalize by the time the Client destructor ran. (PR [#6293](https://github.com/realm/realm-core/pull/6293), since v13.4.1)
 
-### Breaking changes
```

Comment:
This should probably not be removed

Reply:
oh, sorry I trusted the merge, let me fix it ..